themes/src/main/resources/theme/keycloak.v2/login/buttons.ftl

```diff
@@ -1,13 +1,16 @@
-<#macro actionGroup>
+<#macro actionGroup horizontal=false>
```

Comment:
We have the possibility to have action group items sorted horizontally or vertically. 

Useful for buttons.